site/source/user-manual/reset-password.rst

```diff
@@ -0,0 +1,46 @@
+.. _reset-password:
+
+===================
+Reset Your Password
+===================
+
+.. contents::
+  :depth: 2
+  :local:
+
+#. Obtain the latest copy of EmbassyOS with your product key included, by `downloading it from our image server <https://images.start9.com/>`_.
+#. Flash the downloaded image to your microSD card, using `this guide <../flashing>`_.
```

Comment:
Probably should use absolute references, rather than relative, but it does work fine